docs/developer/issue-triage.md

```diff
@@ -0,0 +1,72 @@
+    * An answer to the author's problem was proposed using existing functionality
+  * These issues should be tagged `waiting-for-author` in addition to any other categorizing tags (`bug`, `enhancement`, etc)
+* Waiting for codeowner
+  * Some issues related to community components within Alloy will be dependent on the community maintainer of the component, and other issues may be dependent on codeowners of components that are not maintained by additional teams within Grafana (beyla, pyroscope, database_observability)
+  * These issues should be tagged `waiting-for-codeowner` in addition to any other triage that is initially possible
+  * These issues should retain the `needs-triage` label until the codeowner has responded
+* Ready to implement/fix/document
+  * An issue is ready to implement or fix if the scope is well understood, and if it is an issue it should be replicatable
+  * These issues should be tagged `bug`, `enhancement`, `type/docs`, or `flaky-test`
+  * If the issue should be in the next release, it should be tagged `release-blocker`
+  * If an attempt is being made to internally prioritize the issue at Grafana, it should be tagged `backlog`. If there is a time or release commitment associated with this decision it should be in a comment on the issue.
+  * If the issue is a good candidate for a first time contributor or another interested community member, it should be tagged `good first issue`
+  * If the issue is a good candidate for a larger investment by an interested community member, it should be tagged `help wanted`
+  * *These issues should no longer have the `needs-triage` label*
+* Closed
+  * An issue might be closed after triage if
+    * a solution was offered, the issue was labelled `waiting-for-author`, and the author confirmed their need was met
+    * there is an existing duplicate issue with sufficient context to resolve the issue
+    * the issue was already solved (there should be a duplicate closed issue in most cases, link to it)
+    * based on discussion, the issue should be re-opened as a new `proposal` based on concensus in the issue comments
+  * It's unlikely an issue will be closed after first triage, unless it doesn't meet community standards.
+
+### Stale issues
+
+The `needs-attention` label is applied to issues that are seen as stale in a github action.
+This includes issues that have not been interacted with in 90 days.
+Issues with the `needs-attention` label may be closed if they are not in an actionable state.  
+
+The `keepalive` label can be applied to exempt an issue or pull request from being marked as stale.
+
+### Additional labels
+
+There are a variety of other labels that can be applied to issues and pull requests to help provide context to the issue. Wherever possible, relevant labels should be applied.
+
+* The `component-request` label can help identify requests for new components.
+* The `os:windows` should be used when changes are relevant to the Windows OS.
```

Comment:
We definitely have a few conventions label-naming-wise. What do you think about consolidating around slashes and kebab-case?

`component/request` or `component/new✨`
`os/windows`
`status/keepalive`
`v2/breaking-change`
`help-wanted`
`good-first-issue`

And probably other's I'm missing

Reply:
I like your overall ideas on label standardization, IMO that's not in the scope of this PR but I'm in favor of doing it. I'll leave existing ones as previously used in this PR, then we can update in a follow up as needed.